spec/date_filter_spec.rb

```diff
@@ -124,8 +124,8 @@
     assert_equal nil, filtered
     assert_equal :nils, errors
   end
-  
-  it "considers empty strings to be empty" do
+
+  it "considers empty strings to invalid" do
```

Comment:
We're no longer changing the behaviour of `:empty`, so I think this should stay as it was before.

Reply:
Reverted